mod/web_services/views/default/object/api_key.php

```diff
@@ -43,11 +43,27 @@
 	$content .= elgg_format_element('div', ['class' => ['hidden', 'webservices-secret-' . $entity->guid]], $secret);
 }
 
+// imprint
+$imprint = [];
+
+if ($entity->isKeyActive()) {
+	$imprint[] = [
+		'icon_name' => 'check',
+		'content' => elgg_echo('status:enabled'),
+	];
+} else {
+	$imprint[] = [
+		'icon_name' => 'ban',
+		'content' => elgg_echo('status:disabled'),
+	];
+}
+
 $params = [
 	'title' => $entity->getDisplayName(),
 	'byline' => false,
 	'access' => false,
 	'content' => $content,
+	'imprint' => $imprint,
```

Comment:
if imprint was rewritten to ternary notation, you can add it directly here

Reply:
don't agree